src/pages/Books.jsx

```diff
@@ -3,12 +3,10 @@ import BookList from '../components/BookList';
 import AddBook from '../components/AddBook';
 
 const Books = () => {
-  return (
-    <>
-      <BookList />
-      <AddBook />
-    </>
-  );
+  <>
+    <BookList />
+    <AddBook />
+  </>;
```

Comment:
Hello my friend, So close.

- [ ] To return the component without declaring any function above. React to return ask you to abbreviate the return something like this:

```javascript
const Books = () => (
  <>
    <BookList />
    <AddBook />
  </>
);
```

😉